hudi-flink/src/main/java/org/apache/hudi/sink/partitioner/profile/DeltaWriteProfile.java

```diff
@@ -59,7 +59,7 @@ public DeltaWriteProfile(HoodieWriteConfig config, HoodieFlinkEngineContext cont
       List<FileSlice> allSmallFileSlices = new ArrayList<>();
       // If we can index log files, we can add more inserts to log files for fileIds including those under
       // pending compaction.
-      List<FileSlice> allFileSlices = fsView.getLatestFileSlicesBeforeOrOn(partitionPath, latestCommitTime.getTimestamp(), true)
+      List<FileSlice> allFileSlices = fsView.getLatestFileSlicesBeforeOrOnForFlink(partitionPath, latestCommitTime.getTimestamp(), true)
           .collect(Collectors.toList());
       for (FileSlice fileSlice : allFileSlices) {
```

Comment:
Thanks, let's just make this flag as false, just like what `SparkUpsertDeltaCommitPartitioner` does: https://github.com/apache/hudi/blob/4e8accc179d8f60c874d748167018196900b7d12/hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/deltacommit/SparkUpsertDeltaCommitPartitioner.java#L95

Reply:
got it!